auth_backends/pipeline.py

```diff
@@ -62,43 +49,22 @@ def update_email(strategy, details, user=None, *args, **kwargs):  # pylint: disa
         #    True if usernames don't match, False if they match.
         set_custom_attribute('update_email.username_mismatch', username_mismatch)
```

Comment:
This is highly unexpected, so let's drop this. I think we can leave the warning, but update its text. I'll add another comment.